### PR TITLE
test: added new page objects and pages to ssr test

### PR DESCRIPTION
**Pull request by Ivan-Vidiaiev**
closes: https://spryker.atlassian.net/browse/HRZ-2467

### Checklist before requesting a code review

- [x] PR title is aligned with [PR titles and conventional commits](https://spryker.atlassian.net/wiki/spaces/FA/pages/3821175291/PR+titles+and+conventional+commits)
- [x] I have performed a self-review of my code and have fixed all jobs failures

### CODE REVIEW

File: apps/storefront-e2e/src/integration/search.cy.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { LandingPage } from '../support/page_objects/landing.page';
 import { ProductDetailsPage } from '../support/page_objects/product-details.page';
+import { SearchPage } from '../support/page_objects/search.page';
```
Comment (AndreyYevtukhov) [inline apps/storefront-e2e/src/integration/search.cy.ts]: Import is not used, let's remove it

File: apps/storefront-e2e/src/integration/search.cy.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 import { ProductStorage } from '../test-data/product.storage';
+import { SearchStorage } from '../test-data/search-query';
```
Comment (AndreyYevtukhov) [inline apps/storefront-e2e/src/integration/search.cy.ts]: Import is not used, let's remove it

File: apps/storefront-e2e/src/integration/ssr.cy.ts [hunk 3]
```diff
@@ -19,6 +52,8 @@ describe('SSR suite', { tags: 'smoke' }, () => {
 
     pdp.visit();
 
+    verifyHeader();
+
```
Comment (AndreyYevtukhov) [inline apps/storefront-e2e/src/integration/ssr.cy.ts:59]: Let's make spacing between `page.visit();` and `verifyHeader() / verifyFooter()` consistent in all tests, cause now it is hard to read :)

File: apps/storefront-e2e/src/integration/ssr.cy.ts [hunk 4]
```diff
@@ -30,26 +65,67 @@ describe('SSR suite', { tags: 'smoke' }, () => {
...
+    searchPage.getFacets().should('be.visible');
+    searchPage.getProductSort().should('be.visible');
+    searchPage.getOryxCards().should('have.length.greaterThan', 1);
+    verifyFooter();
+  });
+
+  it('must render Category page', () => {
+    const categoryId = '6';
+    const categoryData = { id: categoryId };
```
Comment (AndreyYevtukhov) [inline apps/storefront-e2e/src/integration/ssr.cy.ts]: We can drop `categoryId` here

```suggestion
    const categoryData = { id: '6' };
```

**Review by AndreyYevtukhov — CHANGES_REQUESTED** [review]
Looks good, only a couple of small things here

File: apps/storefront-e2e/src/integration/search.cy.ts (hunk 2 above)
Comment (Ivan-Vidiaiev) [reply apps/storefront-e2e/src/integration/search.cy.ts]: Done ✅

File: apps/storefront-e2e/src/integration/search.cy.ts (hunk 1 above)
Comment (Ivan-Vidiaiev) [reply apps/storefront-e2e/src/integration/search.cy.ts]: Done ✅

File: apps/storefront-e2e/src/integration/ssr.cy.ts (hunk 4 above)
Comment (Ivan-Vidiaiev) [reply apps/storefront-e2e/src/integration/ssr.cy.ts]: Done ✅

File: apps/storefront-e2e/src/integration/ssr.cy.ts (hunk 3 above)
Comment (Ivan-Vidiaiev) [reply apps/storefront-e2e/src/integration/ssr.cy.ts:59]: Done ✅

**Review by AndreyYevtukhov — APPROVED** [review]

**Pull request merged**